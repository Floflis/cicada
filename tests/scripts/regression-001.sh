export FOO=foo
echo "A${FOO}B$FOO/C${FOO}D"
echo "A$FOO/B${FOO}C$FOO/D"
