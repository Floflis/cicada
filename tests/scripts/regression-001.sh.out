AfooBfoo/CfooD
Afoo/BfooCfoo/D
